rust/hyperlane-base/src/server/server.rs

```diff
@@ -86,7 +86,12 @@ mod tests {
             Arc::new(CoreMetrics::new("test", 8080, mock_registry).unwrap()),
         );
         let server = Arc::new(server);
-        let _run_server = server.run(vec![]).await;
+        // Run the server in the background
+        let _server_task = tokio::spawn(async move {
+            if let Err(err) = server.run(vec![]).await {
```

Comment:
it's usually recommended to just unwrap in unit tests so it's easier to see where things go wrong